.github/workflows/build-assets-on-release.yml

```diff
@@ -0,0 +1,72 @@
+name: Build assets on release and create archive
+on: 
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+          node: [14]
+
+    steps:
+    - name: 'Checkout master branch'
+      uses: actions/checkout@master
+
+    - name: PHP setup
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: 7.4
+        coverage: pcov
+
+    - name: Get composer cache directory
+      id: composer-cache
+      run: |
+        echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+    - name: Cache composer dependencies
+      uses: actions/cache@v1
+      with:
+        path: ${{ steps.composer-cache.outputs.dir }}
+        key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-composer-
+    - name: Install composer packages
+      run: composer install --no-progress
```

Comment:
You can replace this with 

```yaml
# Install dependencies and handle caching in one go.
# @link https://github.com/marketplace/actions/install-composer-dependencies
- name: Install Composer dependencies
   uses: ramsey/composer-install@v1
```

Since the composer is in the root of the project